core/src/proxy/block_engine_stage.rs

```diff
@@ -72,13 +73,10 @@ pub struct BlockBuilderFeeInfo {
     pub block_builder_commission: u64,
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
 pub struct BlockEngineConfig {
-    /// Address to the external auth-service responsible for generating access tokens.
-    pub auth_service_endpoint: Endpoint,
-
-    /// Primary backend endpoint.
-    pub backend_endpoint: Endpoint,
+    /// Block Engine URL
+    pub block_engine_url: String,
```

Comment:
removing auth service endpoint might break some current users of block engine that're still providing the ports on the CLI. probably fine for v1.15/v1.16, but not okay for v1.14 since ideally we don't introduce breaking changes in patch versions

Reply:
makes sense, will change on a major version boundary

i just noticed it had been deprecated for a while and assumed everyone had switched over

Reply:
can you add to PR notes for future that this is removed in this PR?